### ISSUE TITLE
City links not interactive via keyboard

### ISSUE THREAD

**Issue by chriswilty** [ECMWFCode4Earth/vAirify#350]
Keyboard navigation within the grid works fine, but interactive elements inside grid cells by default cannot be reached. They can be clicked by mouse, but are not triggered when hitting Enter, thus we are excluding people who cannot use a mouse.

There are examples of managing focus within cells in the AgGrid docs, but nothing about how to allow events to propagate to child elements. It's pretty annoying that AgGrid hasn't implemented any built-in config setting to allow keyboard events to propagate, but we can do it ourselves:
https://github.com/ag-grid/ag-grid/issues/4114#issuecomment-1517089053

Idea is to add a keyboard listener to the enclosing grid cell element for the Enter key, while allowing mouse click events to work as normal. With a useEffect, we can add the listener on mount and remove it on unmount. Simples... though I've not tried it out yet 🍀
